ldes-server-infra-mongo/mongo-admin-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/mongo/view/repository/DataServiceEntityRepository.java

```diff
@@ -2,6 +2,9 @@
 
 import be.vlaanderen.informatievlaanderen.ldes.server.admin.mongo.view.entity.DataServiceEntity;
 import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
 
 public interface DataServiceEntityRepository extends MongoRepository<DataServiceEntity, String> {
+    @Query(value = "{'_id':  {$regex:  ?0}}", delete = true)
```

Comment:
This query can give problems if the name of a eventstream happens to contain the name of a different eventstream, for example: deleting the eventstream Observations would also delete all dcats of the views of eventstream QualityObservations.